modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -536,19 +594,24 @@ return view.extend({
 			uci.unset('dhcp', section, 'dns');
 		};
 
-		so = ss.option(form.Value, 'mac', _('MAC address'));
-		so.datatype = 'list(macaddr)';
+		var macwildcard = function(multicast) {
+			var m = this.value.match(/^([a-fA-F0-9]{2}|\*):([a-fA-F0-9]{2}:|\*:){4}(?:[a-fA-F0-9]{2}|\*)$/);
+			return this.assert(m != null && !(+m[1] & 1) == !multicast,
```

Comment:
`this.assert(...)` cannot be used in this context, neither is the `multicast` parameter available. 

Should be something like:

```javascript
so.validate = function(section_id, value) {
    var macaddrs = L.toArray(value);

    for (var i = 0; i < macaddrs.length; i++)
        if (!macaddrs[i].match(/^([a-fA-F0-9]{2}|\*):([a-fA-F0-9]{2}:|\*:){4}(?:[a-fA-F0-9]{2}|\*)$/))
            return _('Expecting a valid MAC address, optionally including wildcards');;

    return true;
};
```

Reply:
👍